Makefile

```diff
@@ -65,6 +65,9 @@ build-xcframework:
 	ditto -c -k -X --rsrc --keepParent Carthage/Sentry.xcframework Carthage/Sentry.xcframework.zip
 	ditto -c -k -X --rsrc --keepParent Carthage/Sentry-Dynamic.xcframework Carthage/Sentry-Dynamic.xcframework.zip
 	ditto -c -k -X --rsrc --keepParent Carthage/SentrySwiftUI.xcframework Carthage/SentrySwiftUI.xcframework.zip
+	mv Carthage/Sentry.xcframework.zip Sentry.xcframework.zip
+	mv Carthage/Sentry-Dynamic.xcframework.zip  Sentry-Dynamic.xcframework.zip
+	mv Carthage/SentrySwiftUI.xcframework.zip SentrySwiftUI.xcframework.zip
```

Comment:
The `mv` destination arguments could just be the destination args in the `ditto` commands

Reply:
Couldn't modify unchanged lines in a suggestion comment, so i made a PR: https://github.com/getsentry/sentry-cocoa/pull/3807